app/models/customer.rb

```diff
@@ -0,0 +1,7 @@
+class Customer < ApplicationRecord
+
+  has_many :reservations, dependent: :destroy
```

Comment:
let's discuss all the dependents in this PR

Reply:
If we are destroying a customer, we can safely destroy their reservations too.